From 81f37b958e6398c34aba71c8813396b48b7e8d53 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Fri, 1 Mar 2019 14:20:43 +0800
Subject: [PATCH] Revert "arm64: dts: rockchip: add cpu-avs node for rk3399"

This reverts commit 28e5496a4b30d78231370cb4b5a45779bcec1c4e.
According to:
commit b0005b79e440ecb007231a80e887389f88941f9d
Author: Finley Xiao <finley.xiao@rock-chips.com>
Date:   Wed Apr 12 18:33:32 2017 +0800

    arm64: dts: rockchip: delete cpu-avs device node

    Change-Id: I86dd02761a4156768af018c0c90a61afb0ff74a6
    Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>

Change-Id: Ibe50ddd33280dcaba0af835bc3d6c0dc4a24b003
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 25 ------------------------
 1 file changed, 25 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index b0d82fe9b747..bdfc99bbed4e 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -182,31 +182,6 @@
 		method = "smc";
 	};
 
-	cpu_avs: cpu-avs {
-		cluster0-avs {
-			cluster-id = <0>;
-			min-volt = <800000>; /* uV */
-			min-freq = <408000>; /* KHz */
-			leakage-adjust-volt = <
-			/*  mA        mA         uV */
-			    0         254        0
-			>;
-			nvmem-cells = <&cpul_leakage>;
-			nvmem-cell-names = "cpu_leakage";
-		};
-		cluster1-avs {
-			cluster-id = <1>;
-			min-volt = <800000>; /* uV */
-			min-freq = <408000>; /* KHz */
-			leakage-adjust-volt = <
-			/*  mA        mA         uV */
-			    0         254        0
-			>;
-			nvmem-cells = <&cpub_leakage>;
-			nvmem-cell-names = "cpu_leakage";
-		};
-	};
-
 	timer {
 		compatible = "arm,armv8-timer";
 		interrupts = <GIC_PPI 13 IRQ_TYPE_LEVEL_LOW 0>,
-- 
2.35.3

